src/platforms/vue-native/scripts/transformerPlugin.js

```diff
@@ -29,22 +29,23 @@ if (reactNativeMinorVersion >= 59) {
   }
 }
 
-function sourceMapAstInPlace(tsMap, babelAst) {
-  return SourceMapConsumer.with(tsMap, null, consumer => {
-    traverse.cheap(babelAst, node => {
-      if (node.loc) {
-        const originalStart = consumer.originalPositionFor(node.loc.start)
-        if (originalStart.line) {
-          node.loc.start.line = originalStart.line
-          node.loc.start.column = originalStart.column
-        }
-        const originalEnd = consumer.originalPositionFor(node.loc.end)
-        if (originalEnd.line) {
-          node.loc.end.line = originalEnd.line
-          node.loc.end.column = originalEnd.column
-        }
+function sourceMapAstInPlace(sourceMap, babelAst) {
+  const consumer = new SourceMapConsumer(sourceMap)
+
+  traverse.cheap(babelAst, node => {
+    if (node.loc) {
+      const originalStart = consumer.originalPositionFor(node.loc.start)
+      // Removed the column mapping with original position as we have only mapped lines during generation
```

Comment:
@rahul-mahato We should leave the column mapping as it is unless it is causing issues.

You can add your thoughts in this thread